### PR TITLE
Feature/370 support persistence

### PR DESCRIPTION
**Pull request by subigre**
Pull request regarding ehrbase/project_management#370

### CODE REVIEW

File: src/test/resources/Observation/observation-bloodpressure-example.json [hunk 1]
```diff
@@ -47,12 +47,12 @@
     "text": "Blood pressure systolic & diastolic"
   },
   "subject": {
-    "reference": "Patient/example"
+    "reference": "urn:uuid:example"
```
Comment (ppazos) [inline src/test/resources/Observation/observation-bloodpressure-example.json:50]: @subigre please set a UUID like 07f602e0-579e-4fe3-95af-381728bf0d49 to "example"

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 2]
```diff
@@ -442,7 +442,7 @@ public void createBloodPressure() throws IOException {
         String resource = getContent("classpath:/Observation/observation-bloodpressure-example.json");
 
         // Change patients id to test patient id
-        resource = resource.replaceAll("Patient/example", "Patient/" + this.subjectIdValue);
+        resource = resource.replaceAll("urn:uuid:example", "urn:uuid:" + this.subjectIdValue);
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:445]: @subigre please change the "example" matching to the uuid format matching to keep all the tests similar.

File: src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java [hunk 3]
```diff
@@ -92,8 +92,8 @@ public void createDiagnoseCondition() throws IOException {
 
         String resource = getContent("classpath:/Condition/condition-example.json");
         resource = resource.replaceAll(
-                "Patient/([0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12})",
-                "Patient/" + this.subjectIdValue);
+                "urn:uuid:([0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12})",
```
Comment (ppazos) [inline src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:95]: @subigre please bring indentation back 4 spaces (same for the tests below)

**Review by ppazos — CHANGES_REQUESTED** [review]
@subigre see small changes requested.

**Comment by ppazos** [comment]
@subigre please also check the conflict with develop branch.

**Comment by subigre** [comment]
Dear @ppazos, I did the requested changes

**Review by ppazos — APPROVED** [review]
Looks good :shipit:

**Pull request merged**